opencti-platform/opencti-graphql/src/migrations/1678266052390-roles-under-groups.js

```diff
@@ -0,0 +1,45 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { logApp } from '../config/conf';
+import { findAll, findRoleById, userAddRelation, userIdDeleteRelation } from '../domain/user';
+import { addGroup, groupAddRelation } from '../domain/group';
+
+export const up = async (next) => {
```

Comment:
Migration should be done in 3 steps;

1. Create group from role and create the relation
2. Iter on user and assign them on the right group
3. Delete all relations from user to roles

+ Migration name must be changed to go after the last one got from the rebase

Reply:
+ Auto assign on role must create a group with auto assign

Reply:
Hum, interesting point, it could lead to strange behavior, must be added in documentation

Reply:
done